### PR TITLE
QAE-297 - E2E test case for related post meta

### PR DESCRIPTION
**Pull request by SimranChougule1**
### Description

Added test case for related post meta

### Screenshots

https://share.getcloudapp.com/xQuzp9Yn

### How has this been tested?

Run single test: npm run test:e2e:interactive tests/e2e/specs/customizer/blog/single-post/related-post-meta.test.js

### Checklist:
- [ ] My code is tested
- [ ] My code passes the PHPCS tests
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [ ] I've added proper labels to this pull request

### CODE REVIEW

File: tests/e2e/specs/customizer/blog/single-post/related-post-meta.test.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import { createURL, createNewPost, publishPost } from '@wordpress/e2e-test-utils';
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/blog/single-post/related-post-meta.test.js:1]: need to change "publish-post" util

File: tests/e2e/specs/customizer/blog/single-post/related-post-meta.test.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+import { setCustomize } from '../../../../utils/customize';
+describe( 'Single post font option under the customizer', () => {
+	it( 'related post meta option should apply correctly', async () => {
+		const relatedpostMeta = {
```
Comment (bsfsnehal) [inline tests/e2e/specs/customizer/blog/single-post/related-post-meta.test.js:5]: constant name should be in camelCase

File: tests/e2e/specs/customizer/blog/single-post/related-post-meta.test.js (hunk 1 above)
Comment (SimranChougule1) [reply tests/e2e/specs/customizer/blog/single-post/related-post-meta.test.js]: Done

File: tests/e2e/specs/customizer/blog/single-post/related-post-meta.test.js (hunk 2 above)
Comment (SimranChougule1) [reply tests/e2e/specs/customizer/blog/single-post/related-post-meta.test.js]: Done

**Review by bsfsnehal — APPROVED** [review]

**Pull request merged**